Source/WebInspectorUI/UserInterface/Views/AuditTestCaseContentView.css

```diff
@@ -36,14 +36,14 @@
     top: 0;
     z-index: var(--z-index-header);
     background-color: var(--audit-test-header-background-color);
-    -webkit-backdrop-filter: blur(20px);
+    -webkit-backdrop-filter: blur(1.25em);
```

Comment:
We do a background color average (effectively an infinite gaussian) on iOS native with reduced transparency, but I don't recall having thought about making the blur radius relative to the font size. Cool idea!